### PR TITLE
Let's try this again

### PR DESCRIPTION
**Pull request by Hyperfresh**
Refer to #24

### CODE REVIEW

**Comment by Hyperfresh** [comment]
Fixed conflicts as per e3a3923

**Comment by Hyperfresh** [comment]
** seems i need to fix `##[error]src/index.ts(356,22): error TS2304: Cannot find name 'img'.` - will do in a sec

**Comment by Hyperfresh** [comment]
ok now im confused, there's no code in those lines, hmm lemme investigate more

**Comment by Hyperfresh** [comment]
`'ejs' refers to a UMD global, but the current file is a module. Consider adding an import instead.` 

module? hmm

**Comment by Hyperfresh** [comment]
'img' still broken but ejs does. Still investigating.

**Comment by iwa** [comment]
hey! thanks for the PR c:
you don't have to create a new PR in order to change the target branch, I can just change it by myself dw

i'm gonna resolve the conflicts
it comes because I still haven't merged `dev` to `master` yet, and as I made a lot of big changes, it's doing some kind of shit

File: src/commands/utility/color.ts [hunk 1]
```diff
@@ -39,15 +39,12 @@ module.exports.run = async (bot:Client, msg:Message, args:string[], db:Db) => {
...
+                  "description": "You're now wearing the <@&${colors[choice]}> color!\nType `?color off` to take off the color.",
                   "color": 11675878,
                   "author": {
                     "name": msg.author.username,
                     "icon_url": msg.author.avatarURL({ format: 'png', dynamic: false, size: 128 })
-                  },
-                  "footer": {
-                      "text": "type `?color off` to take off the color"
-                  }
+                  } // Got rid of footer because code blocs (``) don't work in embed footers. Moved to "description". - Hy~
```
Comment (iwa) [inline src/commands/utility/color.ts:47]: i know, it was just to "highlight" the command, but it'll be easier to see in desc than footer yeah

**Review by iwa — APPROVED** [review]
fixed the things that were wrong + deleted some changes

if you want to keep a custom README on your repo, i think you should create a new branch with only the readme, and make this branch default, so it won't include it in your PRs
cause otherwise i'll have to always override it as I did today

**Comment by Hyperfresh** [comment]
Ah k, I’ll have to investigate how to set that all up correctly later ^ ^

**Comment by iwa** [comment]
alrighty, thanks for the PR anyways c:

**Pull request merged**